app/api/admin.py

```diff
@@ -24,7 +24,7 @@ def give_existing_user_admin_privileges(request: Request, id: str, token: Access
     if member["role"] == Role.admin:
         raise HTTPException(400, "User already admin")
     
-    results = db.members.find_one_and_update({'id': member["id"]}, {'$set': {'role': f'{Role.admin}'}})
+    results = db.members.find_one_and_update({'id': member["id"]}, {'$set': {'role': f'{Role.admin.value}'}})
```

Comment:
Preferably not, but it seems support was dropped for using that syntax on format strings. The code before the chage evaluates to the string `"Role.admin"` instead of the desired `"admin"`.

Reply:
I see, maybe we can `use_enum_values`?